redis/asyncio/cluster.py

```diff
@@ -958,17 +960,18 @@ def __eq__(self, obj: Any) -> bool:
 
     _DEL_MESSAGE = "Unclosed ClusterNode object"
 
-    def __del__(self) -> None:
+    def __del__(
+        self, _warn: Any = warnings.warn, _grl: Any = asyncio.get_running_loop
```

Comment:
Can we split these lines to too (how did black pass this!). I found it.. very difficult to find where __grl_ was defined - so IMHO there's an ergonomic issue here. I fully accept that this is a nit + preference.

Reply:
sure.   This is a pattern seen in the standard lib, the idea is to capture some methods into the function so that they are available even during module teardown.  Pretty ugly.

Reply:
I can't split the args, black formats it straight back to one line :(

Reply:
you need to add comma after last arg to make black put them on separate lines. same for list, tuples etc. final comma splits line